### DISCUSSION TITLE
Coolify - Gotenberg Service Template

### DISCUSSION THREAD

**Discussion by nktnet1** [gotenberg/gotenberg#1079] [Q&A] [answered]
A Coolify template for Gotenberg was requested here:
- https://github.com/coollabsio/coolify/discussions/2488

This would be an awesome addition if you have time to work on it :).

Documentations for creating a new template service are available here:
- https://coolify.io/docs/contribute/service

---

Additionally, a template for Dokploy would also be great!

**Comment by nktnet1** [comment]
Made PR for Coolify here: https://github.com/coollabsio/coolify/pull/4759

**Reply by nktnet1** [reply]
Made PR for Dokploy here: https://github.com/Dokploy/dokploy/pull/1065

**Reply by nktnet1** [reply]
Made PR for awesome-selfhosted-data here: https://github.com/awesome-selfhosted/awesome-selfhosted-data/pull/1090

**Reply by nktnet1** [reply, marked as answer]
All 3 pull requests have been merged 🚀

**Reply by gulien** [reply]
Thanks @nktnet1 🙏
